contextual_lenses/tape_gfp/tape_gfp_one_hot_lenses_parallel.py

```diff
@@ -0,0 +1,223 @@
+"""Contextual lenses on TAPE fluoresence dataset."""
```

Comment:
As above, I would look to remove this file and just have one for tape_gfp. You can have a flag for whether to pmap or not.